packages/core/src/shared/plugins/createQueryCacheToState.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  type AncestorOf,
+  type ENode,
+  type TEditor,
+  type TNodeEntry,
+  type Value,
+  getAboveNode,
+  getNodeEntries,
+  getNodeEntry,
+} from '@udecode/slate';
+import { getBlockAbove } from '@udecode/slate-utils';
+
+import type { PlateEditor } from '../types/PlateEditor';
+
+import { createPluginFactory } from '../utils/createPluginFactory';
+
+export type QueryCachToSateEditor<V extends Value> = {
+  state: {
+    aboveEntries?: Generator<TNodeEntry<ENode<V>>, void, undefined>;
```

Comment:
let's reuse the same naming convention than Slate queries. For example `Transforms.above` -> `above`